backend/app/controllers/development/users_controller.rb

```diff
@@ -7,12 +7,48 @@ class Development::UsersController < ApplicationController; end
 
 class Development::UsersController < ApplicationController
   MOCK_USERS = [
-    { user_id: '00000000-0000-0000-0000-000000000001', role: 'admin', name: 'Admin Uno' },
-    { user_id: '00000000-0000-0000-0000-000000000001', role: 'researcher', name: 'Researcher Uno' },
-    { user_id: '00000000-0000-0000-0000-000000000002', role: 'user', name: 'User Uno' },
-    { user_id: '00000000-0000-0000-0000-000000000003', role: 'user', name: 'User Dos' },
-    { user_id: '00000000-0000-0000-0000-000000000004', role: 'user', name: 'User Tres' },
-    { user_id: '00000000-0000-0000-0000-000000000005', role: 'user', name: 'User Cuatro' }
+    {
+      user_id: '00000000-0000-0000-0000-000000000001',
+      role: 'admin',
+      first_name: 'Admin',
+      last_name: 'Uno',
+      name: 'Admin Uno'
+    },
+    {
+      user_id: '00000000-0000-0000-0000-000000000001',
+      role: 'researcher',
+      name: 'Researcher Uno',
+      first_name: 'Researcher',
+      last_name: 'Uno'
+    },
+    {
+      user_id: '00000000-0000-0000-0000-000000000002',
+      role: 'user',
+      name: 'User Uno',
+      first_name: 'User',
+      last_name: 'Uno'
+    },
+    {
+      user_id: '00000000-0000-0000-0000-000000000003',
+      role: 'user',
+      name: 'User Dos',
+      first_name: 'User',
+      last_name: 'Dos'
+    },
+    {
+      user_id: '00000000-0000-0000-0000-000000000004',
+      role: 'user',
+      name: 'User Tres',
+      first_name: 'User',
+      last_name: 'Tres'
+    },
+    {
+      user_id: '00000000-0000-0000-0000-000000000005',
+      role: 'user',
+      name: 'User Cuatro',
+      first_name: 'User',
+      last_name: 'Cuatro'
+    }
   ].freeze
```

Comment:
hrmm.. I find the earlier version much easer to read. is there a reason you expanded all of them?

Reply:
it was getting a bit long on each line (~125 chars with the leading whitespace) i will revert if needed, maybe we can add a max line length lint rule in the future?